### PR TITLE
Use latest wallet package

### PR DESCRIPTION
**Pull request by popenta**
Integrated `sdk-py-wallet v0.8.0` into mxpy.

### CODE REVIEW

File: multiversx_sdk_cli/delegation/staking_provider.py [hunk 1]
```diff
@@ -52,9 +52,7 @@ def prepare_args_for_add_nodes(args: Any):
         pem_file = ValidatorPEM.from_file(validator_pem)
 
         validator_signer = ValidatorSigner(pem_file.secret_key)
-        message = ArbitraryMessage(bytes.fromhex(account.address.hex()))
-
-        signed_message = validator_signer.sign(message).hex()
+        signed_message = validator_signer.sign(account.address.pubkey).hex()
```
Comment (andreibancioiu) [inline multiversx_sdk_cli/delegation/staking_provider.py:55]: :+1: here we do not use `MessageV1`, correct.

File: pyproject.toml [hunk 2]
```diff
@@ -27,7 +28,7 @@ dependencies = [
   "requests-cache",
   "rich==13.3.4",
   "multiversx-sdk-network-providers>=0.11.0",
-  "multiversx-sdk-wallet>=0.7.0",
+  "multiversx-sdk-wallet>=0.8.0",
```
Comment (andreibancioiu) [inline pyproject.toml:31]: Here, we should protect against breaking changes, and add upper bounds, as well. Perhaps for all our libraries?

In requirements.txt, as well.

File: requirements.txt [hunk 3]
```diff
@@ -9,6 +9,6 @@ semver
 requests-cache
 rich==13.3.4
 
-multiversx-sdk-core==0.6.0
-multiversx-sdk-network-providers>=0.11.0
-multiversx-sdk-wallet>=0.8.0
+multiversx-sdk-core>=0.6.0,<0.7.0
+multiversx-sdk-network-providers>=0.11.0,<0.12.0
+multiversx-sdk-wallet>=0.8.0,<0.9.9
```
Comment (andreibancioiu) [inline requirements.txt:14]: Typo `0.9.0`.

File: pyproject.toml (hunk 2 above)
Comment (popenta) [reply pyproject.toml]: Done for our libraries

**Review by andreibancioiu — APPROVED** [review]

**Review by ssd04 — APPROVED** [review]

**Pull request merged**